pkg/mediationcontainer/remote_mediation_client.go

```diff
@@ -117,7 +117,7 @@ func (remoteMediationClient *remoteMediationClient) Init(probeRegisteredMsgCh ch
 				// stop server messages listener
 				remoteMediationClient.stopMessageHandler()
 				// Reconnect
-				err := transport.Connect()
+				err := transport.Connect(jwtToken)
```

Comment:
My concern is that during a reconnect (i.e., after Turbo server restart), is the `jwtToken` still valid? Is there a need to negotiate with the server to refresh the token here?

Reply:
This is a good point, we should refresh the token.